.golangci.example.yml

```
@@ -449,6 +449,16 @@ linters-settings:
     servingv1: knative.dev/serving/pkg/apis/serving/v1
     # using `autoscalingv1alpha1` alias for `knative.dev/serving/pkg/apis/autoscaling/v1alpha1` package
     autoscalingv1alpha1: knative.dev/serving/pkg/apis/autoscaling/v1alpha1
+  gomoddirectives:
+    # Allow local `replace` directives. Default is false.
+    replace-local: false
+    # List of allowed `replace` directives. Default is empty.
+    replace-allow-list:
```

would nesting not be easier to maintain:
```suggestion
    replace:
      # Allow local `replace` directives. Default is false.
      local: false
      # List of allowed `replace` directives. Default is empty.
      allow-list:
```

---

I don't want to do that

---

out of curiosity, why?